src/__tests__/webpack-plugin.test.ts

```diff
@@ -0,0 +1,131 @@
+import webpack from 'webpack';
+import { ReactLooselyLazyPlugin, getBundleFiles } from '../webpack';
+import path from 'path';
+
+const manifestFilename = 'rll-manifest.json';
+const config = {
+  entry: {
+    main: path.join(__dirname, '__fixtures__/webpack/app/index.js'),
+  },
+  output: {
+    path: path.join(__dirname, '__fixtures__/webpack/output'),
+    filename: '[name].js',
+    chunkFilename: '[name].js',
+    publicPath: '/output/',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            babelrc: false,
+            presets: [
+              ['@babel/preset-env', { modules: false }],
+              '@babel/preset-react',
+            ],
+          },
+        },
+      },
+    ],
+  },
+  resolve: {
+    alias: {
+      test: path.resolve(__dirname, 'webpack'),
+    },
+  },
+  plugins: [
+    new ReactLooselyLazyPlugin({
+      filename: manifestFilename,
+    }),
+  ],
+};
+
+describe('plugin functionality', () => {
+  let manifest;
```

Comment:
Do we need `let manifest` here? Why not inline it in the single test below

Reply:
Ah I think I was going to use the var in another test but I'm not doing that now so fine to change to const